### PR TITLE
Fixed height on top leaderboard

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/CONCF-84

Doesn't show ad-slot when noAds is set.

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
Why do we have those merge pull here?

File: front/templates/main/application.hbs [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {{smart-banner isVisible=view.smartBannerVisible}}
 {{site-head smartBannerVisible=view.smartBannerVisible sideNavCollapsed=view.sideNavCollapsed trackClick='trackClick'}}
 {{side-nav isCollapsed=view.sideNavCollapsed}}
-{{ad-slot name='MOBILE_TOP_LEADERBOARD' noAds=noAds}}
+{{#unless noAds}}
```
Comment (hakubo) [inline front/templates/main/application.hbs]: not displaying ads is handled inside the component so we don't have to do it here.

Comment (rogatty) [inline front/templates/main/application.hbs]: We're always rendering HTML for it. With `noads=1` it's:
```
<div id="MOBILE_TOP_LEADERBOARD" class="ad-slot" style="display: none !important;"></div>
```

Do you think it should be rendered and hidden?

Comment (hakubo) [inline front/templates/main/application.hbs]: yes, that was requested by ads team, this way they can always fully debug what is going on with ads

Comment (rogatty) [inline front/templates/main/application.hbs]: Actually, good point. We check `noAds` in `AdSlotComponent.ts` and condition logic is more complicated than `if (noAds)`. This shouldn't be done in template.

Comment (dianafa) [inline front/templates/main/application.hbs]: aaaaa, ok!

**Comment by kenkouot** [comment]
Yeah, can we just redo this branch without all the merges

**Pull request closed without merging**